NodeKit/NodeKit/Layers/RequestBuildingLayer/URLQueryInjectorNode.swift

```diff
@@ -1,39 +1,39 @@
 import Foundation
 
-/// Ошибки для узла `URLQueryInjectorNode`
+/// Errors for ``URLQueryInjectorNode``
 public enum URLQueryInjectorNodeError: Error {
-    /// Возникает в случае, если не удалось создать URLComponents из URL
+    /// Occurs if URLComponents couldn't be created from the URL.
```

Comment:
FYI: в офф текстах принято писать без сокращений
```suggestion
    /// Occurs if URLComponents could not be created from the URL.
```